internal/pkg/describe/service.go

```diff
@@ -33,13 +33,17 @@ type ecsClient interface {
 	TaskDefinition(taskDefName string) (*ecs.TaskDefinition, error)
 }
 
-type configStoreSvc interface {
+// ConfigStoreSvc wraps methods of config store.
+type ConfigStoreSvc interface {
 	GetEnvironment(appName string, environmentName string) (*config.Environment, error)
 	ListEnvironments(appName string) ([]*config.Environment, error)
+	ListServices(appName string) ([]*config.Service, error)
 }
 
-type deployStoreSvc interface {
+// DeployStoreSvc wraps methods of deploy store.
+type DeployStoreSvc interface {
```

Comment:
_thinking out loud_:

I wonder if there is a way of decoupling this interface name from the struct (`deploy.Store`) that implements it.   
The methods here return `([]string, error)` so any pkg should be able to implement these methods (which is great), whereas the `ConfigStoreSvc` methods return `*config` types. So I don't mind that interface to be prefixed with `Config` so much.

Reply:
maybe: `DeployedEnvServicesLister` 💭